[{"ContainerToolsException.Message":"Error response from daemon: Cannot kill container: 47ae1090f8d12f11de8b6cfb60ce9144c0b880ec49a080f5d774fca694d84382: Container 47ae1090f8d12f11de8b6cfb60ce9144c0b880ec49a080f5d774fca694d84382 is not running","ContainerToolsException.ErrorCode":"DT1001","ContainerToolsException.StackTrace":"   em System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   em System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   em Microsoft.Docker.DockerComposeClient.<ExecuteAsync>d__13.MoveNext()\r\n--- Fim do rastreamento de pilha do local anterior onde a exceção foi gerada ---\r\n   em System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   em System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   em Microsoft.Docker.DockerComposeClient.<DownAsync>d__8.MoveNext()\r\n--- Fim do rastreamento de pilha do local anterior onde a exceção foi gerada ---\r\n   em System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   em System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   em Microsoft.Docker.DockerWorkspace.<ComposeDownContainersAsync>d__31.MoveNext()\r\n--- Fim do rastreamento de pilha do local anterior onde a exceção foi gerada ---\r\n   em System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   em System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   em Microsoft.Docker.BuildTasks.DockerBuildTask.<ExecuteAsync>d__38.MoveNext()","ContainerToolsException.HResult":"-2146233088","ContainerToolsException.CommandLine.Program":"docker-compose","ContainerToolsException.CommandLine.TelemetrySafeArguments":"","ContainerToolsException.CommandLine.ExitCode":"1","ContainerToolsException.CommandLine.ErrorOutput":"Container dockercompose6252227391999846379_db_1  Killing\r\nContainer dockercompose6252227391999846379_api_1  Killing\r\nContainer dockercompose6252227391999846379_api_1  Error while Killing\r\nContainer dockercompose6252227391999846379_db_1  Error while Killing\r\nError response from daemon: Cannot kill container: 47ae1090f8d12f11de8b6cfb60ce9144c0b880ec49a080f5d774fca694d84382: Container 47ae1090f8d12f11de8b6cfb60ce9144c0b880ec49a080f5d774fca694d84382 is not running"}]